build.ps1

```diff
@@ -5,21 +5,21 @@ New-Item -ItemType Directory -Force -Path ./nuget
 dotnet tool restore
 
 pushd ./src/Storage
-./build.ps1 $args
+Invoke-Expression "./build.ps1 $args"
```

Comment:
Without this, `$args` is passed as a single quote-wrapped value to `build.ps1`, so multiple options/targets cannot be passed.